widget/embedded/src/components/TokenList/TokenList.styles.ts

```diff
@@ -0,0 +1,30 @@
+import { styled } from '@rango-dev/ui';
+
+export const Title = styled('div', {
+  display: 'flex',
+  alignItems: 'center',
+});
+export const Content = styled('div', {
+  minHeight: 250,
+});
+
+export const Tag = styled('div', {
```

Comment:
the color of the tag and its background aren't matched with Figma. Also please check that it is centered.

Reply:
The colors are changing